### ISSUE TITLE
Allow to create small edges

### ISSUE THREAD

**Issue by palvarezlopez** [eclipse-sumo/sumo#13833]
useful for JuPedSim mode

**Comment by namdre** [comment]
If it's mainly used for JuPedSim, maybe we should only activate it in JuPedSim mode? 

As it stands I find the feature somewhat limited in its applicability (and thus wasteful of menta/screen space). Yes, it creates short edges but only If I happen to want edges that are exactly 5 m long and horizontal.  If I need anything else, I have to use move mode to shorten/rotate the edge. However, this exact workflow is already possible with the normal edge creation mode.

**Comment by palvarezlopez** [comment]
yes, an specific case for JuPedSim mode could be a good solution.

What do you think @behrisch @bcoueraud87 @RNippold ?

**Comment by bcoueraud87** [comment]
As discussed with @behrisch once, these waypoints could also be modelled somehow as POIs (which are visually more satisfying in netedit), and then the small edges could be created automatically. In any case, it's true that it is only JuPedSim-related at that time, so it should go to the JuPedSim mode.
